jaxrs-clients/build.gradle

```diff
@@ -30,4 +33,17 @@ dependencies {
     testCompile libs.hamcrest
     testCompile libs.junit
     testCompile libs.mockwebserver
+
+    processor libs.immutables
+}
+
+[configurations.integrationTestsCompile, configurations.integrationTestsCompileClasspath, configurations.integrationTestsCompileOnly, configurations.integrationTestsRuntime].each {
```

Comment:
verified (via Gradle :dependencies task) that these configurations have jackson 2.4 only